### PR TITLE
Feature/#218 add user athentication to update a project for student endpoint

### PR DESCRIPTION
**Pull request by begoIT24**

### CODE REVIEW

**Comment by begoIT24** [comment]
Thanks for you help and contribution @ivanlegranbizarro

File: app/Http/Controllers/api/Student/UpdateStudentProjectController.php [hunk 1]
```diff
@@ -4,31 +4,40 @@
...
     {
-        $this->studentupdateProjectService = $studentupdateProjectService;
+        $this->updateStudentProjectService = $updateStudentProjectService;
+        $this->studentService = $studentService;
     }
 
     public function __invoke(UpdateStudentProjectRequest $request, $studentId, $projectId): JsonResponse
     {       
         try {
+            $userProfile = $this->studentService->findUserByStudentID($studentId);
```
Comment (jordimorillo) [inline app/Http/Controllers/api/Student/UpdateStudentProjectController.php:29]: Passport check is redundant when the authorisation is required via a route middleware inclusion

File: app/Http/Controllers/api/Student/UpdateStudentProjectController.php [hunk 2]
```diff
@@ -13,18 +13,18 @@
 
 class UpdateStudentProjectController extends Controller
 {
-    private $studentUpdateProjectService;
+    private $studentupdateProjectService;
```
Comment (jordimorillo) [inline app/Http/Controllers/api/Student/UpdateStudentProjectController.php]: $studentUpdateProjectService

**Review by jordimorillo — APPROVED** [review]

File: app/Http/Controllers/api/Student/UpdateStudentProjectController.php (hunk 1 above)
Comment (begoIT24) [reply app/Http/Controllers/api/Student/UpdateStudentProjectController.php]: done

File: app/Http/Controllers/api/Student/UpdateStudentProjectController.php (hunk 2 above)
Comment (begoIT24) [reply app/Http/Controllers/api/Student/UpdateStudentProjectController.php]: corrected in a posterior commit

**Review by ivanlegranbizarro — APPROVED** [review]

**Pull request merged**